### PR TITLE
Observability language tweaks

### PR DESCRIPTION
**Pull request by keithhc2**
### Check List
- [x] Commits are signed per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: _observability-plugin/app-analytics.md [hunk 1]
```diff
@@ -6,58 +6,60 @@ nav_order: 80
...
 - color: The color of the availability badge on the home page.
 - name: The text in the availability badge on the home page.
-- expression: Comparison operator to determine the availability. 
+- expression: Comparison operator to determine the availability.
 - value: Value to use when calculating availability.
 
 ![Configuring availability]({{site.url}}{{site.baseurl}}/images/app_availability_level.gif)
 
-By default, Application analytics shows results from the last 24 hours of your data. To see data from a different timeframe, use the date and time selector.
+By default, application analytics shows results from the last 24 hours of your data. To see data from a different timeframe, use the date and time selector.
```
Comment (lizsnyder) [inline _observability-plugin/app-analytics.md:52]: Time frame

File: _observability-plugin/app-analytics.md [hunk 2]
```diff
@@ -6,58 +6,60 @@ nav_order: 80
...
 
 #### Time series metric
 
-A time series metric is any visualization that has a query that spans over a timestamp and is a line chart. Users can use the power of the PPL language to define arbitrary conditions on their logs to create a visualization over time.
+A time series metric is any visualization that has a query that spans over a timestamp and is a line chart. You can then use PPL to define arbitrary conditions on their logs to create a visualization over time.
```
Comment (lizsnyder) [inline _observability-plugin/app-analytics.md:56]: on your logs

File: _observability-plugin/event-analytics.md [hunk 3]
```diff
@@ -34,22 +34,22 @@ To save a visualization, expand the save dropdown menu next to **Refresh**, ente
 
 ## View logs
 
-Users have several ways to view their logs.
+The following are methods you can use to view logs.
 
 ### Trace log correlation
 
-Users who regularly track events across applications have the capability to correlate logs and traces. To view the correlation, users have to index the traces as per the Open Telemetry standards (similar to Trace analytics). Once users add a **TraceId** field to their logs, they can view the trace information that corresponds to the same execution context as the log.
+If you regularly track events across applications, you can correlate logs and traces. To view the correlation, you have to index the traces according to Open Telemetry standards (similar to trace analytics). Once you add a `TraceId` field to your logs, your can view the correlated trace information in the event explorer log details. This method lets you correlate logs and traces that correspond to the same execution context.
```
Comment (lizsnyder) [inline _observability-plugin/event-analytics.md:41]: "you can view" (not your can view)

File: _observability-plugin/event-analytics.md [hunk 4]
```diff
@@ -34,22 +34,22 @@ To save a visualization, expand the save dropdown menu next to **Refresh**, ente
...
+If you prefer watching events happen live, you can configure an interval so event analytics automatically refreshes the content. Live tail lets you stream logs live to OpenSearch observability event analytics based on the provided PPL query, as well as provide rich functionality such as filters. Doing so improves your debugging experience and lets you monitor your logs in real-time without having to manually refresh. You can also choose intervals and switch  between them to dictate how often live tail should stream live logs. This feature is similar to the CLI's `tail -f` command in that it only retrieves the most recent live logs by possibly eliminating a large portion of live logs. Live tail also provides you with the total count of count of live logs received by OpenSearch during the live stream, which you can use to better understand the incoming traffic.
```
Comment (lizsnyder) [inline _observability-plugin/event-analytics.md:53]: nit: extra space between "switch" and "betwen"

also, "total count of count of"

**Review by Naarcha-AWS — APPROVED** [review]

**Review by lizsnyder — APPROVED** [review]

**Pull request merged**